### ISSUE TITLE
ML waardebepalingsmodel trainen

### ISSUE THREAD

**Issue by vdeb27** [vdeb27/woningzoeker#20]
## Beschrijving
Train een machine learning model voor nauwkeurigere waardebepalingen.

## Huidige situatie
- Regelgebaseerd model in `backend/services/valuation.py`
- Basis m² prijs × oppervlakte + correcties
- Werkt redelijk, maar kan nauwkeuriger

## Te implementeren
1. **Data verzamelen**
   - Historische transacties (Kadaster, na issue #2)
   - Features: oppervlakte, bouwjaar, energielabel, buurt, woningtype
   - Target: verkoopprijs

2. **Model training**
   ```python
   from sklearn.ensemble import GradientBoostingRegressor
   from sklearn.model_selection import cross_val_score
   
   features = ['oppervlakte', 'bouwjaar', 'energielabel_encoded', 
               'buurt_gem_prijs', 'woningtype_encoded']
   
   model = GradientBoostingRegressor(n_estimators=100)
   model.fit(X_train, y_train)
   ```

3. **Feature engineering**
   - Buurt statistieken als features
   - Afstand tot centrum/OV
   - Tijdseffecten (seizoen, trend)

4. **Evaluatie**
   - RMSE, MAE, R²
   - Cross-validatie per buurt
   - Feature importance analyse

5. **Integratie**
   - Model opslaan (joblib/pickle)
   - Laden bij startup
   - Fallback naar regelgebaseerd

## Dependencies
```
pip install scikit-learn joblib pandas
```

## Bestanden
- `backend/models/ml_valuation.py` - Model training
- `backend/services/valuation.py` - Integratie
- `data/models/valuation_model.joblib` - Trained model

## Acceptatiecriteria
- [ ] Model getraind op beschikbare data
- [ ] R² > 0.7 op test set
- [ ] Feature importance rapport
- [ ] Confidence intervals per schatting
